src/planning/plan/tests/planning-result-test.ts

```diff
@@ -10,57 +10,71 @@
 import {assert} from '../../../platform/chai-web.js';
 import {Recipe} from '../../../runtime/recipe/recipe.js';
 import {Search} from '../../../runtime/recipe/search.js';
+import {Manifest} from '../../../runtime/manifest.js';
 import {Relevance} from '../../../runtime/relevance.js';
-import {PlanningTestHelper} from '../../testing/planning-test-helper.js';
+import {Runtime} from '../../../runtime/runtime.js';
 import {PlanningResult} from '../../plan/planning-result.js';
 import {Suggestion} from '../../plan/suggestion.js';
+import {FakeSlotComposer} from '../../../runtime/testing/fake-slot-composer.js';
+import {storageKeyPrefixForTest} from '../../../runtime/testing/handle-for-test.js';
+import {StubLoader} from '../../../runtime/testing/stub-loader.js';
+import {StrategyTestHelper} from '../../testing/strategy-test-helper.js';
 
 describe('planning result', () => {
   async function testResultSerialization(manifestFilename) {
-    const helper = await PlanningTestHelper.createAndPlan({manifestFilename});
-    assert.isNotEmpty(helper.suggestions);
-    helper.suggestions.forEach(s => {
-      s.relevance = Relevance.create(helper.arc, s.plan);
-      s.relevance.apply(new Map([[s.plan.particles[0], [1]]]));
-    });
-    const result = new PlanningResult(helper.envOptions);
-    result.merge({suggestions: helper.suggestions}, helper.arc);
+    const loader = new StubLoader({});
+    const context = await Manifest.load(manifestFilename, loader);
+    const runtime = new Runtime(loader, FakeSlotComposer, context);
+    const arc = runtime.newArc('demo', storageKeyPrefixForTest());
+    const suggestions = await StrategyTestHelper.planForArc(arc);
+
+    assert.isNotEmpty(suggestions);
+    // suggestions.forEach(s => {
+    //   s.relevance = Relevance.create(arc, s.plan);
+    //   s.relevance.apply(new Map([[s.plan.particles[0], [1]]]));
+    // });
```

Comment:
Is this something that needs to be revisited?

Reply:
removed.